File: Beginner/Average1/main.go
package main

import (
	"fmt"
)

func main() {

	/**
	 * Escreva a sua solução aqui
	 * Code your solution here
	 * Escriba su solución aquí
	 */
	var a float32
	fmt.Scan(&a)
	var b float32
	fmt.Scan(&b)
	average := ((a * 3.5) + (b * 7.5)) / 11
	fmt.Printf("MEDIA = %.5f\n", average)
}

File: Beginner/Average1/Main.java
import java.io.IOException;
import java.util.Scanner;
 
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        /**
         * Escreva a sua solução aqui
         * Code your solution here
         * Escriba su solución aquí
         */
         
        Scanner sc = new Scanner(System.in);
        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double average = ((a * 3.5) + (b * 7.5)) / 11;
        System.out.printf("MEDIA = %.5f\n", average);

        sc.close();
    }
 
}